infra/gp-grafana-instance/templates/02-system-datasource.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: system-datasource
+  namespace: {{ .Release.Namespace }}
+spec:
+  datasources:
+    - access: proxy
+      editable: true
+      isDefault: true
+      jsonData:
+        httpHeaderName1: Authorization
+        httpMethod: POST
+      secureJsonData:
```

Comment:
Auch hier die Dokumentation bitte anpassen: https://gepardec.atlassian.net/wiki/spaces/G/pages/2460352513/Grafana
Es wird in ArgoCD ignoriert? https://github.com/gepaplexx/gepardec-run-cluster-configuration/pull/8/files#diff-15d4eec1bd3af847278d220199f134a2a7b3bf4b5fbce58685ddac21b9940662R232